From 397c35b1a626e9da6c8338018b41246aee3a2d2d Mon Sep 17 00:00:00 2001
From: Jan Engelhardt <jengelh@inai.de>
Date: Mon, 6 Jul 2015 17:17:20 +0200
Subject: [PATCH 48/62] search: show blank tab for new tab page

Avoid loading the search page everytime, just show a blank one instead.

References: https://github.com/iridium-browser/iridium-browser/issues/32
---
 chrome/browser/search/search.cc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/chrome/browser/search/search.cc b/chrome/browser/search/search.cc
index c3a68e4d757a1..e3b4158b4cc40 100644
--- a/chrome/browser/search/search.cc
+++ b/chrome/browser/search/search.cc
@@ -170,6 +170,7 @@ struct NewTabURLDetails {
       : url(url), state(state) {}
 
   static NewTabURLDetails ForProfile(Profile* profile) {
+    return NewTabURLDetails(GURL("about:blank"), NEW_TAB_URL_VALID);
     // Incognito has its own New Tab.
     if (profile->IsOffTheRecord())
       return NewTabURLDetails(GURL(), NEW_TAB_URL_INCOGNITO);
-- 
2.25.1

